README.md

```diff
@@ -95,3 +95,20 @@ See
 [CONTRIBUTING.md](https://github.com/hashicorp/packer/blob/master/.github/CONTRIBUTING.md)
 for best practices and instructions on setting up your development environment
 to work on Packer.
+
+## Unmaintained Plugins
```

Comment:
Just wondering here: Could 'unmaintained' be too strong of a statement ? Could we say 'community-supported' like we already do ? ( basically meaning we will merge your stuff but not actively maintain it ? ). At the same time it is nice to have a difference between community maintained and not maintained at all...